.github/workflows/ci.yml

```diff
@@ -60,3 +60,50 @@ jobs:
           if [ ! -d docs/doxygen/ ]; then exit 0; fi
           doxygen docs/doxygen/config.doxyfile 2>&1 | tee doxyoutput.txt
           if [[ "$(wc -c < doxyoutput.txt | bc)" = "0" ]]; then exit 0; else exit 1; fi
+  spell-check:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout Parent Repo
+          uses: actions/checkout@v2
+          with:
+            ref: master
+            repository: aws/aws-iot-device-sdk-embedded-C
+        - run: rm -r libraries/standard/coreJSON
```

Comment:
`rm -rf`?

Reply:
This script was copied from coreMQTT, but I don't think adding the -f will hurt. I have added it.